json_schema/core/file/file_core.json

```
@@ -64,7 +64,8 @@
                 "DCP/1 Matrix Service",
                 "LungMAP",
                 "Zenodo",
-                "Publication"
+                "Publication",
+                "HCA Ingest"
```

Having second thought on this. Should this be

```suggestion
                "DCP/2 Ingest"
```

---

… considering the existing `DCP/1 Matrix Service`, that is.

---

That sounds good to me, I will commit the change!